src/org/usfirst/frc4904/robot/humaninterface/operators/DefaultOperator.java

```diff
@@ -28,11 +29,11 @@ public void bindCommands() {
 		RobotMap.Component.operatorStick.button4.onlyWhileHeld(new BallioOuttake());
 		RobotMap.Component.operatorStick.button5.onlyWhileHeld(new BallioFloorClear());
 		RobotMap.Component.operatorStick.button6.onlyWhileHeld(new BallioCycle());
-		new ThresholdCommand(new GearioIntake(), RobotMap.Component.operatorStick::getY, DefaultOperator.INTAKE_THRESHOLD)
-			.start();
-		new ThresholdCommand(new GearioOuttake(), RobotMap.Component.operatorStick::getY, -DefaultOperator.INTAKE_THRESHOLD,
-			true)
-				.start();
+		RobotMap.Component.teensyStick.getButton(15).whenPressed(new CalibrateCameraExposure());
```

Comment:
I will have to talk to Nikhil about this, I believe that he has the master list of buttons, but he was the one that set it to fifteen, so it should be correct.